deploy-manage/production-guidance/kibana-task-manager-scaling-considerations.md

```diff
@@ -56,36 +54,36 @@ For detailed troubleshooting guidance, see [Troubleshooting](../../troubleshoot/
 
 ::::
 
-
-
 ## Deployment considerations [_deployment_considerations]
 
 {{es}} and {{kib}} instances use the system clock to determine the current time. To ensure schedules are triggered when expected, synchronize the clocks of all nodes in the cluster using a time service such as [Network Time Protocol](http://www.ntp.org/).
 
-
 ## Scaling guidance [task-manager-scaling-guidance]
 
 How you deploy {{kib}} largely depends on your use case. Predicting the throughput a deployment requires to support Task Management is difficult because features can schedule an unpredictable number of tasks at a variety of scheduled cadences.
 
 However, there is a relatively straight forward method you can follow to produce a rough estimate based on your expected usage.
 
-
 ### Default scale [task-manager-default-scaling]
 
-By default, {{kib}} polls for tasks at a rate of 10 tasks every 3 seconds. This means that you can expect a single {{kib}} instance to support up to 200 *tasks per minute* (`200/tpm`).
+By default, {{kib}} polls for tasks at a rate of 10 tasks every 500 milliseconds. This means that you can expect a single {{kib}} instance to support up to 1200 _tasks per minute_ (`1200/tpm`).
+
+- As of v8.15 a new task claim strategy was introduced: `mget`. And as of v8.18, `mget` has been made the default strategy -as part of some performance improvement efforts-, with a default polling interval of 500 milliseconds. Since these changes offers a better task execution performance, It is highly recommended you to upgrade to v8.18.
+
+- The strategy used can be changed by using the `xpack.task_manager.claim_strategy` setting. Valid values are: `mget` and `update_by_query`
 
-In practice, a {{kib}} instance will only achieve the upper bound of `200/tpm` if the duration of task execution is below the polling rate of 3 seconds. For the most part, the duration of tasks is below that threshold, but it can vary greatly as {{es}} and {{kib}} usage grow and task complexity increases (such as alerts executing heavy queries across large datasets).
+- Maximum number of concurrent tasks can be changed by using `xpack.task_manager.capacity`, The default value is 10, the minimum and maximum values ​​are 5 and 50 respectively.
+
+In practice, a {{kib}} instance will only achieve the upper bound of `1200/tpm` if the duration of task execution is below the polling rate of 500 milliseconds. For the most part, the duration of tasks is -below?- that threshold, but it can vary greatly as {{es}} and {{kib}} usage grow and task complexity increases (such as alerts executing heavy queries across large datasets).
```

Comment:
I wonder if we need to start changing our wording that in general tasks will take longer than the poll interval to complete and the task duration needs to become part of the calculation (perhaps we already do somewhere and just need to reference it).

```suggestion
In practice, a {{kib}} instance will only achieve the upper bound of `1200/tpm` if the duration of task execution is below the polling rate of 500 milliseconds. For the most part, the duration of tasks is below that threshold, but it can vary greatly as {{es}} and {{kib}} usage grow and task complexity increases (such as alerts executing heavy queries across large datasets).
```